apps/src/templates/teacherDashboard/RosterDialog.jsx

```diff
@@ -0,0 +1,154 @@
+import React from 'react';
+import BaseDialog from '../BaseDialog';
+import color from '../../util/color';
+import locale from '@cdo/locale';
+
+const styles = {
+  title: {
+    position: 'absolute',
+    left: 20,
+    color: color.dark_charcoal,
+    margin: '15px 0',
+  },
+  content: {
+    position: 'absolute',
+    left: 20,
+    top: 50,
+    right: 20,
+    bottom: 70,
+    overflowY: 'scroll',
+  },
+  classroomRow: {
+    padding: 10,
+    cursor: 'pointer',
+  },
+  highlightRow: {
+    backgroundColor: color.default_blue,
+    color: color.white,
+  },
+  footer: {
+    position: 'absolute',
+    bottom: 15,
+    right: 20,
+    left: 20,
+  },
+  buttonPrimary: {
+    float: 'right',
+    background: color.orange,
+    color: color.white,
+    border: '1px solid #b07202',
+    borderRadius: 3,
+    boxShadow: 'inset 0 1px 0 0 rgba(255, 255, 255, 0.63)',
+    fontSize: 14,
+    padding: '8px 20px',
+  },
+  buttonSecondary: {
+    float: 'left',
+    background: '#eee',
+    color: '#5b6770',
+    border: '1px solid #c5c5c5',
+  },
+};
+
+const ClassroomList = ({classrooms, onSelect, selectedId}) => classrooms.length ?
+  <div>
+    {classrooms.map(classroom => (
+      <div
+        style={Object.assign({},
+          styles.classroomRow,
+          selectedId === classroom.id && styles.highlightRow
+        )}
```

Comment:
note: with Object spread syntax, you should no longer need to use `assign` you can instead do:
```
style={{
  ...styles.classroomRow,
  ...(selectedId === classroom.id && styles.highlightRow)
}}